File: pset6/mario.py
import cs50

def main():
    
    while True:
        print("Height:", end="")
        height = cs50.get_int()
        if height > 0 and height < 24:
            break
    
    spaces = height - 1
    hashes = 2
        
    for i in range(height):
        print(" " * spaces + "#" * hashes + "\n")
        spaces -= 1
        hashes += 1
    
if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    int spaces;
    int hashes;
    
    do 
    {
        printf("height: ");
        height = GetInt();
    }
    while (height < 0 || height > 23);
    
    for (int i = 1; i <= height; i++)
        {
            for (spaces = height-i; spaces != 0; spaces--)
            {
                printf(" ");
            }
            for (hashes = 0; hashes <= i; hashes++)
            {
                printf("#");
            }
            printf("\n");
        }
    
    return 0;
}